frontend/src/components/AddPostForm/AddPostForm.styled.js

```diff
@@ -0,0 +1,61 @@
+import styled from 'styled-components';
+
+export const Form = styled.form`
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+
+  padding: 1em;
+  border-radius: 20px;
+  background-color: #bbc;
+
+  * {
+    margin: 3px 0px;
+  }
+`;
+
+export const FormTitle = styled.h2`
+  height: 30px;
+  margin: 10px;
+
+  color: gray;
+  font-family: 'Roboto', sans-serif;
```

Comment:
Czy to coś zmienia? Roboto jest ustawione globalnie, więc powinno być i tutaj.

Reply:
Wcześniej nie było. Usuwam.